### PR TITLE
[ADD] port trp_web_click2call

### PR DESCRIPTION
**Pull request by hbrunn**

### CODE REVIEW

**Comment by StefanRijnhart** [comment]
When testing, most times I clicked the button whether in readonly or edit mode it did not seem to have any effect. Only once did I manage to trigger the server call, and I don't know how.

The buttons are a bit big for the new style form. Can they be made a bit smaller?

File: trp_web_click2call/static/src/js/click2call.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+        },
+        on_button_clicked: function() {
+            click2call(this);
+        }
+    });
+
+    instance.web.form.widgets.add('click2call', 'openerp.web.form.Click2call');
+
+    instance.trp_web_click2call.click2call = click2call;
+    instance.trp_web_click2call.click2call_number = function(number)
```
Comment (StefanRijnhart) [inline trp_web_click2call/static/src/js/click2call.js:37]: Dead code path?

Comment (hbrunn) [inline trp_web_click2call/static/src/js/click2call.js:37]: No, our customer modules use that in kanban views

**Comment by hbrunn** [comment]
Buttons are resized.

Concerning the event handling: I can't reproduce this, but looking closer at the code it seems a good idea to use ReinitializeFieldMixin's api for setting up the event handler. Maybe that helps?

File: trp_web_click2call/static/src/js/click2call.js (hunk 1 above)
Comment (StefanRijnhart) [inline trp_web_click2call/static/src/js/click2call.js:37]: Add that as a comment. I have a bad memory.

Comment (StefanRijnhart) [inline trp_web_click2call/static/src/js/click2call.js:37]: Merged this and adding this comment directly.

**Comment by StefanRijnhart** [comment]
Thanks for your work. It works as expected now.

**Pull request merged**